File: AtomykPlay/app/src/main/java/com/atomykcoder/atomykplay/adapters/ViewHolders/BlackListViewHolder.java
package com.atomykcoder.atomykplay.adapters.ViewHolders;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.annotation.NonNull;

import com.atomykcoder.atomykplay.R;
import com.atomykcoder.atomykplay.adapters.Generics.GenericViewHolder;

public class BlackListViewHolder extends GenericViewHolder<String> {
    public final ImageView imageView;
    private final TextView textView;

    public BlackListViewHolder(@NonNull View itemView) {
        super(itemView);
        textView = itemView.findViewById(R.id.blacklist_directory_path);
        imageView = itemView.findViewById(R.id.delete_from_blacklist);

    }

    @Override
    public void onBind(String item) {
        textView.setText(item);
    }
}

File: AtomykPlay/app/src/main/java/com/atomykcoder/atomykplay/adapters/viewHolders/BlackListViewHolder.kt
package com.atomykcoder.atomykplay.adapters.viewHolders

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import com.atomykcoder.atomykplay.R
import com.atomykcoder.atomykplay.adapters.generics.GenericViewHolder

class BlackListViewHolder(itemView: View) : GenericViewHolder<String>(itemView) {
    val imageView: ImageView
    private val textView: TextView

    init {
        textView = itemView.findViewById(R.id.blacklist_directory_path)
        imageView = itemView.findViewById(R.id.delete_from_blacklist)
    }

    override fun onBind(item: String) {
        textView.text = item
    }
}
